### PR TITLE
Make compartiment circles clickable

### PR DESCRIPTION
**Pull request by FrankApiyo**
Fixes: #108

### CODE REVIEW

**Comment by FrankApiyo** [comment]
This PR replaces this https://github.com/OpenSRP/opensrp-web/pull/143

**Comment by FrankApiyo** [comment]
[This](https://github.com/OpenSRP/opensrp-web/pull/143#pullrequestreview-337767416l) comment has also been responded to here

File: src/components/DataCircleCard/index.tsx [hunk 1]
```diff
@@ -138,24 +139,26 @@ function DataCircleCard({
...
-                  // tslint:disable-next-line: jsx-no-lambda
-                  onClick={() => {
-                    if (filterArgs) {
-                      addFilterArgsActionCreator(filterArgs);
-                    }
-                  }}
-                >
-                  <span className="number">{spec.riskValue}</span>
-                </Link>
+              <li
```
Comment (moshthepitt) [inline src/components/DataCircleCard/index.tsx]: Why are opting out of using `Link`?

Comment (FrankApiyo) [reply src/components/DataCircleCard/index.tsx]: There really wasn't a good reason for this decision except that the route taken here was the easier route.

@moshthepitt  do you think we should stick with `<Link>` ?

Comment (moshthepitt) [reply src/components/DataCircleCard/index.tsx]: Yes I think we should stick with Link if possible

Comment (FrankApiyo) [reply src/components/DataCircleCard/index.tsx]: @moshthepitt after replacing the `<li>` with `<Link>`, I noticed how this makes the code more declarative as opposed to imperative philosophy of react.

**Pull request merged**